firebase/README.md

```diff
@@ -9,3 +16,17 @@ firebase -P staging deploy
 
 To deploy to the prod environment (covidactnow-prod), run:
 firebase -P prod deploy
+
+# Testing Hosting / Functions
+To test hosting / functions locally:
+
+```
+# Run this from the 'firebase/functions' directory to watch / recompile TypeScript files.
+`npm bin`/tsc -w
+
+# Run this to run the hosting / functions local servers.
+firebase serve
+```
```

Comment:
I tried running this, but it didn't work ... do I need to install Firebase globally? I tried running `yarn` in the top-level directory and in `firebase` with no luck 😢